### ISSUE TITLE
Error: 500 Internal Server Error when bootstrapping CCES v8.1.3

### ISSUE THREAD

**Issue by DamaniN** [rubrikinc/terraform-aws-rubrik-cloud-cluster-elastic-storage#8]
**Expected Behavior**

Deploying and bootstrapping CCES v8.1.3 would work, properly

**Current Behavior**

@SpikeyWeasel reported the following:

>  I’ve switched our Rubrik Terraform module to use the latest 8.1 AMI 8.1.3-24772 (rubrik-mp-cc-8-1-3-24772-0d848ae0-5c6d-4949-add5-dd31ea972343 - ami-010f7870292bfdb54) and during the bootstrap phase I’m getting an error as below:
> 
> ``` 
> module.rubrik-cluster.module.rubrik_cluster.rubrik_bootstrap_cces_aws.bootstrap_rubrik_cces_aws: Still creating... [17m50s elapsed]
> module.rubrik-cluster.module.rubrik_cluster.rubrik_bootstrap_cces_aws.bootstrap_rubrik_cces_aws: Still creating... [18m0s elapsed]
> ╷
> │ Error: 500 Internal Server Error
> │
> │   with module.rubrik-cluster.module.rubrik_cluster.rubrik_bootstrap_cces_aws.bootstrap_rubrik_cces_aws,
> │   on .terraform/modules/rubrik-cluster.rubrik_cluster/[main.tf](http://main.tf/) line 235, in resource "rubrik_bootstrap_cces_aws" "bootstrap_rubrik_cces_aws":
> │  235: resource "rubrik_bootstrap_cces_aws" "bootstrap_rubrik_cces_aws" {
> │
> ```
> 
> It however does work with 8.1.1-p1-24326. This is using the “main” branch of your Terraform CC ES module and the Rubrik Provider 2.2.0

**Failure Information (for bugs)**

 ``` 
 module.rubrik-cluster.module.rubrik_cluster.rubrik_bootstrap_cces_aws.bootstrap_rubrik_cces_aws: Still creating... [17m50s elapsed]
 module.rubrik-cluster.module.rubrik_cluster.rubrik_bootstrap_cces_aws.bootstrap_rubrik_cces_aws: Still creating... [18m0s elapsed]
 ╷
 │ Error: 500 Internal Server Error
 │
 │   with module.rubrik-cluster.module.rubrik_cluster.rubrik_bootstrap_cces_aws.bootstrap_rubrik_cces_aws,
 │   on .terraform/modules/rubrik-cluster.rubrik_cluster/[main.tf](http://main.tf/) line 235, in resource "rubrik_bootstrap_cces_aws" "bootstrap_rubrik_cces_aws":
 │  235: resource "rubrik_bootstrap_cces_aws" "bootstrap_rubrik_cces_aws" {
 │
 ```

**Steps to Reproduce**

Deploy the module as documented using CDM v8.1.3-24772 

**Context**

N/A

**Failure Logs**

None Needed.

**Comment by DamaniN** [comment]
After some debugging of the issue, we found that the [Terraform Provider for Rubrik](https://github.com/rubrikinc/terraform-provider-rubrik) was failing while checking the status of the bootstrap operation that had been started. The fix is to retry the status check, however, this change must be made in the [Rubrik Go SDK](https://github.com/rubrikinc/rubrik-sdk-for-go).

**Comment by DamaniN** [comment]
Fixed by rubrikinc/terraform-provider-rubrik#89
